### PR TITLE
add rate limit

### PR DESCRIPTION
**Pull request by Khalid1G**
https://github.com/ISIL-ESTE/Student-Workflow-Organizer/issues/36#issue-1769458696
@bellaabdelouahab 

this need test .

### CODE REVIEW

File: backend-app/app.js [hunk 1]
```diff
@@ -32,12 +32,6 @@ app.use(cors());
 app.use(helmet());
 
 // Limit request from the same API
-const limiter = rateLimit({
```
Comment (bellaabdelouahab) [inline backend-app/app.js:35]: i think you should keep it in app.js it's only 5 line of code

**Comment by bellaabdelouahab** [comment]
and also that doesn't dix the issue of  SSE !

**Comment by Khalid1G** [comment]
I know the solution I think it is easy if we separate routes and apply a rate limit only for the other router ether the SSE for example if we use `/api/v1/...` for the express route we need to use `/` for SSE, with that we make sure the limiter middleware is only applied as expected

**Comment by bellaabdelouahab** [comment]
sure I will be waiting for your pr hahaha.

**Review by bellaabdelouahab — APPROVED** [review]

**Pull request merged**